include/uart.h

```diff
@@ -64,6 +65,62 @@ extern "C" {
  */
 #define UART_ERROR_BREAK    (1 << 3)
 
+/**
+ * @brief UART controller configuration structure
+ *
+ * @param baudrate  Baudrate setting in bps
+ * @param parity    Parity bit, use @ref uart_config_parity
```

Comment:
Any change to fit "enum" in the description, e.g.:

~~~
Parity bit, use enum @ref uart_config_parity
~~~

Oh, and it's probably "parity setting", not "parity bit".